packages/ui-components/src/components/getting-started/getting-started-step-overview.ts

```diff
@@ -0,0 +1,94 @@
+import { BaseElement } from '../../base';
+import { TemplateResult, css, html } from 'lit';
+import { unsafeSVG } from 'lit/directives/unsafe-svg.js';
+import TrailOne from '../../assets/trail-1.svg?raw';
+import TrailTwo from '../../assets/trail-2.svg?raw';
+import TrailThree from '../../assets/trail-3.svg?raw';
+
+export class GettingStartedStepOverview extends BaseElement {
+  static styles = [
+    ...BaseElement.styles,
+    css`
+      .grid-template {
+        grid-template-rows: 250px 250px 390px;
+        grid-template-columns: 1fr 1fr 200px 1fr 1fr;
+      }
+
+      .trail-one {
+        margin: 0 0 -140px 20px;
+      }
+
+      .trail-two {
+        margin: 0 20px -15px 0;
+      }
+
+      .trail-three {
+        margin: 0 0 -110px -80px;
+      }
+    `,
+  ];
+  steps: { title: string; description: TemplateResult }[] = [
```

Comment:
Newline between static and non-static properties

Reply:
fixed in: https://github.com/JortWillemsen/stryker-dashboard-redesign/pull/20